src/state.ts

```diff
@@ -76,11 +76,10 @@ export interface FormStateOptions<M> {
     afterSave?: ValidationOption;
     pauseDuration?: number;
   };
-  isDisabled?: FieldAccessorAllows;
-  isHidden?: FieldAccessorAllows;
-  isReadOnly?: FieldAccessorAllows;
-  isRepeatingFormDisabled?: RepeatingFormAccessorAllows;
-  isRequired?: FieldAccessorAllows;
+  isDisabled?: AccessorAllows;
+  isHidden?: AccessorAllows;
+  isReadOnly?: AccessorAllows;
+  isRequired?: AccessorAllows;
```

Comment:
This can be FieldAllows - more specific. But I think it's okay to use AccessorAllows.